configuration/code_schemes.py

```diff
@@ -10,7 +10,9 @@ def _open_scheme(filename):
 
 
 class CodeSchemes(object):
-    S01E01 = _open_scheme("s01e01.json")
+    DIAGNOSTIC_S01E01 = _open_scheme("diagnostic_s01e01.json")
+    DIAGNOSTIC_S01E02 = _open_scheme("diagnostic_s01e02.json")
+    IMAQAL_COVID19_S01E01 = _open_scheme("s01e01.json")
```

Comment:
I think it would be reasonable to leave this as S01E01, given that this is Project-IMAQAL-COVID19. But no need to change back if you think this helps

Reply:
right, have renamed to S01E01